.travis.yml

```diff
@@ -1,16 +1,29 @@
 language: node_js
-node_js:
-- 10
-- 12
-addons:
-  firefox: latest
-before_script:
-- export DISPLAY=:99.0
-- sh -e /etc/init.d/xvfb start
-- sleep 3 # give xvfb some time to start
-script:
-- npm run lint
-- npm run coverageNode
-- codecov --clear --flags=node
-- npm run coverageWeb
-- codecov --clear --flags=web
+
+matrix:
+  include:
+  - name: node10
+    node_js: 10
+    script:
+    - npm run lint
+    - npm run coverageNode
+
+  - name: node12
+    node_js: 12
+    script:
+    - npm run lint
+    - npm run coverageNode
+
+  - name: firefox
+    node_js: 10
+    addons:
+      firefox: latest
+    before_script:
+    - export DISPLAY=:99.0
+    - sh -e /etc/init.d/xvfb start
+    - sleep 3 # give xvfb some time to start
```

Comment:
This makes me a bit uneasy, but I see this is the recommended way on Trusty (https://docs.travis-ci.com/user/gui-and-headless-browsers/#using-xvfb-directly).

Reply:
It is pretty unsightly, using xvfb-run might be an alternative, but then I'd rather look into running headless Chrome and Firefox somehow. So I'll leave it be :)

Reply:
Gave it a try: https://github.com/GoogleChromeLabs/confluence/pull/405